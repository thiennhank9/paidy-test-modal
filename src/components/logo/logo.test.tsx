import TestRenderer from "react-test-renderer";
import Logo from "./logo";

describe("Logo", () => {
  test("render Logo", () => {
    const testRenderer = TestRenderer.create(<Logo />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});