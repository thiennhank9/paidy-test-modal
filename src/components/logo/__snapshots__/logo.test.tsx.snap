// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo render Logo 1`] = `
<img
  className="logo"
/>
`;
